status/missing_pods_test.go

```diff
@@ -5,7 +5,7 @@ Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
I would leave the indentation as before.

Reply:
Sorry I forgot you mentioned it should be 4 spaces instead of a tab. Updated.

The description is also updated, thanks for reminding me, almost forgot of it.